src/include/uapi/ipc/header.h

```diff
@@ -79,6 +79,7 @@
 #define SOF_IPC_GLB_DAI_MSG			SOF_GLB_TYPE(0x8U)
 #define SOF_IPC_GLB_TRACE_MSG			SOF_GLB_TYPE(0x9U)
 #define SOF_IPC_GLB_GDB_DEBUG                   SOF_GLB_TYPE(0xAU)
+#define SOF_IPC_GLB_LARGE                       SOF_GLB_TYPE(0xBU)
```

Comment:
What is the reason of adding another global message, instead of using `SOF_IPC_GLB_COMP_MSG` with a new `SOF_IPC_COMP_GET/SET_LARGE_DATA` command types?

Reply:
Only thing I see here is that get/set_data is used with controls so currently we can't load the parameters in boot using get/set methods. For EQ it is currently copied from control to component data and sent during component creation. Currently we would need some user space app to load the controls. However. I can look if we could use get/set_ data in component creation.